### PR TITLE
[KYUUBI #2960] TFrontendService.SERVER_VERSION shall be HIVE_CLI_SERVICE_PROTOCOL_V11

### PR DESCRIPTION
**Pull request by CavemanIV**
### _Why are the changes needed?_
1. bugfix for issue #2960

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Review by pan3793 — APPROVED** [review]
LGTM, pending CI

**Review by cxzl25 — APPROVED** [review]

File: kyuubi-common/src/main/scala/org/apache/kyuubi/service/TFrontendService.scala [hunk 1]
```diff
@@ -623,7 +623,7 @@ private[kyuubi] object TFrontendService {
 
   final val CURRENT_SERVER_CONTEXT = new ThreadLocal[FeServiceServerContext]()
 
-  final val SERVER_VERSION = TProtocolVersion.HIVE_CLI_SERVICE_PROTOCOL_V10
+  final val SERVER_VERSION = TProtocolVersion.HIVE_CLI_SERVICE_PROTOCOL_V11
```
Comment (yaooqinn) [inline kyuubi-common/src/main/scala/org/apache/kyuubi/service/TFrontendService.scala:626]: let's use `TProtocolVersion.values.max` once and for all

**Review by yaooqinn — APPROVED** [review]

**Review by turboFei — APPROVED** [review]

**Comment by turboFei** [comment]
thanks, merged to branch-1.5 and master

**Comment by turboFei** [comment]
Hi @CavemanIV 

could you bind the email(zhangliang@trip.com) with your github account?

**Pull request closed without merging**